src/app/modules/item/components/chapter-children/chapter-children.component.ts

```diff
@@ -25,6 +25,8 @@ export class ChapterChildrenComponent implements OnChanges, OnDestroy {
   private reloadData() {
     if (this.itemData.attemptId) {
       this.state = 'loading';
+      if (this.subscription)
+        this.subscription.unsubscribe();
```

Comment:
When not using brackets, inline it. Anyway, in TS, we should use optional chaining.
```suggestion
      this.subscription?.unsubscribe();
```